leftwm-core/src/utils/command_pipe.rs

```diff
@@ -301,6 +302,34 @@ fn build_decrease_main_width(raw: &str) -> Result<Command, Box<dyn std::error::E
     Ok(Command::DecreaseMainWidth(change))
 }
 
+fn build_focus_next_tag(raw: &str) -> Command {
+    match raw {
+        "ignore_empty" | "goto_used" => Command::FocusNextTag {
+            behavior: command::FocusDeltaBehavior::IgnoreEmpty,
+        },
+        "ignore_used" | "goto_empty" => Command::FocusNextTag {
+            behavior: command::FocusDeltaBehavior::IgnoreUsed,
+        },
+        _ => Command::FocusNextTag {
```

Comment:
Not entirely sure, if we shouldn't better match on actual `none | "default"` and error on `_`?
Right now `leftwm-command "FocusNextTag gurke` is behaving like `default`.

Reply:
I missed that this is about command pipe, so obviously `none` is in this case `""`..

Reply:
implemented in newer commits